hw/xbox/dsp/dsp_cpu.c

```diff
@@ -420,23 +429,30 @@ void dsp56k_reset_cpu(dsp_core_t* dsp)
     dsp->disasm_prev_inst_pc = 0xFFFFFFFF;
 }
 
-static OpcodeEntry lookup_opcode(uint32_t op) {
-    OpcodeEntry r = {0};
+static OpcodeEntry* search_opcode(uint32_t op)
+{
+    OpcodeEntry* r;
```

Comment:
In addition to what @JayFoxRox wrote, initialize `r` (like this: `OpcodeEntry* r = NULL;`) to avoid corruption due to potential garbage data.